src/outputtype.cpp

```diff
@@ -20,13 +20,13 @@ static const std::string OUTPUT_TYPE_STRING_BECH32 = "bech32";
 bool ParseOutputType(const std::string& type, OutputType& output_type)
 {
     if (type == OUTPUT_TYPE_STRING_LEGACY) {
-        output_type = OutputType::LEGACY;
+        output_type = OutputType::PKH;
```

Comment:
style-nit: in commit b2da4c74d891fe719b21411f692e3edd4b968c

Do as scripted diff?